kong/plugins/prometheus/exporter.lua

```diff
@@ -54,31 +54,43 @@ local function log(message)
     return
   end
 
-  metrics.status:inc(1, { message.response.status, service_name })
+  local route_name
+  if message and message.route then
+    if message.route.name then
+      route_name = message.route.name
+    else
+      route_name = message.route.id
+    end
```

Comment:
We can simplify this logic as:
```lua
route_name = message.route.name or message.route.id
```

This is a handy way in Lua to to route_name to id when name is nil.